src/main/java/com/adobe/cq/testing/junit/rules/CQUserAgentRule.java

```diff
@@ -0,0 +1,36 @@
+package com.adobe.cq.testing.junit.rules;
```

Comment:
Minor, no copyright header in the file, but could still be relevant, as its Adobe Open Source.